introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-fulltraining.ipynb

```diff
@@ -524,6 +524,17 @@
     "Image(file_name)  "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Evaluation\n",
+    "\n",
+    "Evaluate the image through the network for inteference. The network outputs class probabilities and typically, one selects the class with the maximum probability as the final class output.\n",
+    "\n",
+    "**Note:** The output class detected by the network may not be accurate in this example. This is because we have trained the model only for a couple of epochs. If the network is trained for more epochs (say 20), then the output class will be more accurate."
```

Comment:
Can we change "This is because we have..." to something like "To limit the time and cost of training, we have...".  Just want to make sure the customer understands why we've chosen to only train for 2 epochs.